code/registration.py

```diff
@@ -269,6 +263,171 @@ def load_initial_frames(
     return frames
 
 
+def combine_images_with_individual_titles(image1_path, image2_path, output_path, title1="", title2=""):
+    """
+    Combine two images side-by-side with padding and add individual titles above each image.
+
+    Parameters:
```

Comment:
Can you change the docstrings to numpy docstrings format?

Reply:
Docstrings updated